hera_cal/utils.py

```diff
@@ -150,9 +150,9 @@ def fft_dly(data, df, wgts=None, f0=0.0, medfilt=False, kernel=(1, 11), edge_cut
     return dlys, offset
 
 
-def interp_peak(data, method='quinn'):
+def interp_peak(data, method='quinn', reject_edges=False):
```

Comment:
check whether this works for complex data in the quadratic method, does it need to be abs'ed? Update docstring or put in assertion or something.